iwf.yaml

```diff
@@ -1476,3 +1476,29 @@ components:
           format: int64
         attempt: # it's from Attempt of Cadence/Temporal activity.GetInfo
           type: integer
+    EventType:
+      type: string
+      enum:
+        - WORKFLOW_START_EVENT
+        - WORKFLOW_COMPLETE_EVENT
+        - WORKFLOW_FAIL_EVENT
+    IwfEvent:
+      type: object
+      required:
+        - eventType
+        - workflowType
+        - workflowId
+        - workflowRunId
+      properties:
+        eventType:
+          $ref: '#/components/schemas/EventType'
+        workflowType:
+          type: string
+        workflowId:
+          type: string
+        workflowRunId:
+          type: string
+        stateId:
+          type: string # only applicable for certain events
+        stateExecutionId:
+          type: string # only applicable for certain events
```

Comment:
Maybe also need startTime and endTime(not applicable for start event)?

Reply:
we can then use the endTime to calculate the duration

Reply:
As time (string) or timestamp (int64)? I'm thinking the latter

Reply:
Yeah int64 would be great

Reply:
Done